integration/terraform/ec2/localstack/variables.tf

```diff
@@ -5,7 +5,7 @@ variable "ec2_instance_type" {
 
 variable "ssh_key_name" {
   type    = string
-  default = "cwagent-integ-test-key"
```

Comment:
So we took out SSH access to the EC2s?

Reply:
We have a generated SSH Key if we don't provide any value to this variable `ssh_key_name`. So newcomer is still able to SSH access to the EC2 if they provide their key to this variable and the `ssh_key_value`.

Reply:
This has minimum to get up and running. I can add it back as optional.